lib/src/glam/__init__.py

```diff
@@ -139,6 +146,12 @@ def generate_glycopeptides(
     max_length : int or None, default: None,
         The maximum length peptide to include in the digest output
     semi : bool, default: False,
+    
+     Returns
```

Comment:
Is this indented one space too far?

Reply:
Using something like [numpydoc pre-commit](https://github.com/numpy/numpydoc) or enabling [ruff pydoctsytyle as a pre-commit hook](https://docs.astral.sh/ruff/rules/#pydocstyle-d) would help capture these things.